File: pgm/ABC_Java/ABC_253_A.java
import java.io.*;
// import java.math.*;
import java.util.*;
// import java.util.Map.*;
import java.util.stream.*;

public class Main {
    public static void main (String[] args) throws Exception {
        BufferedReader reader =
            new BufferedReader (new InputStreamReader (System.in));
        final List<Integer> abc = Arrays.stream (reader.readLine().split (" "))
                                      .map (Integer::valueOf)
                                      .collect (Collectors.toList());
        final String Yes = "Yes";
        final String No = "No";
        final int b = abc.get (1);
        Collections.sort (abc);
        System.out.println (b == abc.get (1) ? Yes : No);
        return;
    }
}

File: pgm/ABC_Kotlin/ABC_253_A.kt
// import java.io.BufferedOutputStream
// import java.io.BufferedReader
// import java.io.DataInputStream
// import java.io.FileInputStream
// import java.io.IOException
// import java.io.InputStreamReader
// import java.io.PrintWriter
// import java.lang.StringBuilder
// import java.time.LocalTime
// import java.time.format.DateTimeFormatter
// import java.util.Scanner
// import java.util.StringTokenizer

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val space: String = " "
const val one: Int = 1
const val two: Int = 2

fun main() {
    val abc: IntArray = readln().split(space).map { it.toInt() }.toIntArray()
    val list: List<Int> = abc.sorted()
    val len: Int = list.size
    if (abc[one] == list[len / two]) {
        println(Yes)
    } else {
        println(No)
    }
    kotlin.system.exitProcess(0)
}